src/layouts/components/Editor/extensions/Image/Image.ts

```diff
@@ -11,13 +11,20 @@ interface IsomerImageOptions {
   width?: string
   height?: string
   href?: string
+  style?: string
+}
+
+interface IsomerImageStyle {
+  width?: number
+  backgroundColor?: string
```

Comment:
can delete first! was just raising an example HAHA

Reply:
Sure removed in 103897178315870ae1628dd281b0ef10b6d7f671